client/views/actions/topic_set_by.tpl

```diff
@@ -1,3 +1,4 @@
-Topic set by
+{{translate 'client.topic_set_by.before'}}
 {{> ../user_name}}
-on {{localetime when}}
+{{translate 'client.topic_set_by.between'}} {{localetime when}}
+{{translate 'client.topic_set_by.after'}}
```

Comment:
None of these have a translation key & missing newline

Reply:
This was me deciding if we want to do `{{before}}`, `{{between}}`, `{{after}}`